bound/bound_test.go

```diff
@@ -75,3 +75,45 @@ func TestIsValid(t *testing.T) {
 		t.Error("b4 is invalid")
 	}
 }
+
+func TestUnion(t *testing.T) {
```

Comment:
Please add a TODO at the top of the file to rewrite these tests as table-driven.